content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -215,6 +215,9 @@ kubectl get pods --show-labels
 JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
  && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
 
+# Check which nodes are ready with custom-columns
+kubectl get node -o custom-columns='NODE_NAME:.metadata.name,STAUS:.status.conditions[?(@.type=="Ready")].status'
```

Comment:
```suggestion
kubectl get node -o custom-columns='NODE_NAME:.metadata.name,STATUS:.status.conditions[?(@.type=="Ready")].status'
```

Reply:
The current version is good but Wouldn't it be better to see `Node Age` alongside with status?

Reply:
Thanks a lot for your suggestion, Is it ok to allow this change as it is and latter I will add your suggestion as well. thanks a lot again